rhombus/scribblings/annotation.scrbl

```diff
@@ -65,21 +65,21 @@ the @rhombus(.) operator can be chained for efficient access:
     l1.p2.x
 )
 
-@margin_note_block{Using @rhombus(.) to reach an imported binding, as in
- @rhombus(f2c.fahrenheit_to_celsius), is a different kind of @rhombus(.)
- than the infix expression operator.}
+@margin_note_block{Using @rhombus(., ~datum) to reach an imported binding, as in
+ @rhombus(f2c.fahrenheit_to_celsius, ~datum),
+ is different than the infix expression operator, @rhombus(.).}
 
 More generally, @rhombus(.) access is efficient when the left-hand side
 of @rhombus(.) is an @rhombus(import) prefix, a @tech{namespace}, or
 an expression that can act as a @deftech{dot provider}. A class name
 acts as a namespace to provides access to
-field-accessor functions, as in @rhombus(Posn.x) (which doesn’t get a
-specific @rhombus(x), but produces a function that can be called on a
-@rhombus(Posn) instance to extract its @rhombus(x) field). An identifier
-that is bound using @rhombus(:~) or @rhombus(::) and a class name is
+field-accessor functions, as in @rhombus(#,(@rhombus(Posn, ~datum)).#,(@rhombus(x, ~datum))) (which doesn’t get a
```

Comment:
This is also a dotted name, not a field access.

Reply:
Oh geez.